lib/tasks/populate.rake

```diff
@@ -96,9 +121,9 @@ namespace :pvb do
 
     desc 'populate visits for review apps or dev environments'
     task visits: :environment do
-      %w[Sudbury Pentonville Usk].each do |prison_name|
+      %w[Sudbury Pentonville Usk Aylesbury].each do |prison_name|
         prison = Prison.find_by!(name: prison_name)
-        visits = 0.upto(14).map { |_i|
+        visits = 0.upto(140).map { |_i|
```

Comment:
Could we make the large number of visits optional? For the majority of review apps it wont be necessary.